community/cypher/cypher-compiler-2.3/src/main/java/org/neo4j/cypher/internal/compiler/v2_3/birk/Javac.java

```diff
@@ -99,13 +98,13 @@ public CompilationError( String message )
         return clazz;
     }
 
-    public static InternalExecutionResult newInstance( Class<InternalExecutionResult> clazz, CompletionListener completion, Statement statement,
+    public static InternalExecutionResult newInstance( Class<InternalExecutionResult> clazz, TaskCloser closer, Statement statement,
```

Comment:
This is going to conflict with the profiling PR. I say we merge this one first, and I can rebase the other one.